### PR TITLE
Run Feature Tests in Chrome Headlessly

### PR DESCRIPTION
**Pull request by TylerOSI**
### Jira Story

Indirectly related to

- [Name of Story HOT-2278](https://osi-cwds.atlassian.net/browse/HOT-2278)

## Description
Upgrading our test infrastructure for feature tests to run on the latest stable version of chrome

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
Infrastructure change

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [?] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: docker/base_images/Dockerfile.intake_testing [hunk 1]
```diff
@@ -23,7 +26,8 @@ RUN  apt-get install -y \
     libfontconfig1 \
     libfontconfig1-dev \
     yarn \
-    bzip2
+    bzip2 \
+    man
```
Comment (wmitchell-cambria) [inline docker/base_images/Dockerfile.intake_testing]: Does Jenkins need to check man pages now and then? :-)

File: spec/features/home_page_spec.rb [hunk 2]
```diff
@@ -192,7 +193,7 @@
             expect(find_all('td').last).to have_text('08/17/2018')
           end
           within rows[4] do
-            expect(find_all('td').last).to have_text('01/10/2018')
+            expect(find_all('td').last).to have_text('01/09/2018')
```
Comment (wmitchell-cambria) [inline spec/features/home_page_spec.rb]: I'm hoping that there is not some reason why this change is necessary for the tests to pass, right? Seems harmless...but just double checking that there isn't something weird going on.

File: spec/features/screening/participant/editing_participant_spec.rb [hunk 3]
```diff
@@ -594,7 +595,8 @@
     within edit_participant_card_selector(marge.id) do
       expect(page).to have_field('Approximate Age', disabled: true)
       expect(page).to have_field('Approximate Age Units', disabled: true)
-      fill_in_datepicker 'Date of birth', with: ''
+      find_field('Date of birth').set('', clear: :backspace)
```
Comment (wmitchell-cambria) [inline spec/features/screening/participant/editing_participant_spec.rb]: Oh, interesting... I think this probably fixes a weird bug I came across a while ago around clearing fields.

File: spec/support/capybara.rb [hunk 4]
```diff
@@ -25,6 +24,27 @@
...
+Capybara.register_driver :accessible_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: {
+      args: [
+        'headless',
+        'disable-gpu',
+        'no-sandbox',
+        'disable-dev-shm-usage',
+        'incognito',
+        'window-size=2500,2500'
```
Comment (wmitchell-cambria) [inline spec/support/capybara.rb]: Odd resolution choice. Is this what we already had, or should we target something closer to actual usage? Maybe even a 'real' resolution would look more natural here, like 1920x1200 or 1440x900.

File: docker/base_images/Dockerfile.intake_testing (hunk 1 above)
Comment (TylerOSI) [reply docker/base_images/Dockerfile.intake_testing]: I added this because when i tried to run the chrome executable in docker to get the help menu and see some of the command line options, it gave me a vague error that I couldn't figure out for a while ... until I installed man pages. I think its best to keep it in case we have to do some debugging in docker.

File: spec/features/home_page_spec.rb (hunk 2 above)
Comment (TylerOSI) [reply spec/features/home_page_spec.rb]: I believe it was a miscalculation from UTC to PST

File: spec/features/screening/participant/editing_participant_spec.rb (hunk 3 above)
Comment (TylerOSI) [reply spec/features/screening/participant/editing_participant_spec.rb]: yeah i was happy when i found it. i had to upgrade capybara to use it.
